bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java

```diff
@@ -114,6 +118,11 @@
     private final Logger logger = LoggerFactory.getLogger(LGWebOSTVSocket.class);
     private int nextRequestId = 0;
 
+    private final ScheduledExecutorService scheduler;
+    private @Nullable ScheduledFuture<?> disconnectingJob;
+
+    private static final int DISCONNECTING_DELAY_SECONDS = 2;
```

Comment:
I hate to make you do this as part of this PR but the issue is bad enough and I don't want it to get worse.

Please order the fields by: static final -> final -> non-final.

Reply:
Done for the classes LGWebOSHandler and LGWebOSTVSocket.

Reply:
Please note that the variable logger is very often set at first, even before the static fields. I don't know why and whether it is a good practice. With your request change, this is not the case here.